crates/configuration/src/shared/types.rs

```diff
@@ -365,15 +365,51 @@ pub struct ValidationContext {
 
 #[derive(Default, Debug, Clone, PartialEq, Deserialize)]
 pub struct ChainDefaultContext {
-    pub(crate) default_command: Option<Command>,
```

Comment:
Hi @wirednkod, I'm not fully understand the reason behind this change. This is an internal type that is not exposed through the config builder right?

Reply:
Correct. Since this is internal and as we agreed this we will keep as `pub(crate)`;